### ISSUE TITLE
Torrents are not being added to rtorrent with fast resume

### ISSUE THREAD

**Issue by adamwinn** [mmgoodnow/cross-seed#81]
cross-seed is adding torrents to rtorrent via its inject method, but it is not using fast resume.  Every torrent that is being added is performing a hash check.

**Comment by mmgoodnow** [comment]
Can you check your volume mapping?

From https://github.com/mmgoodnow/cross-seed/wiki/Injection:
> If you use Docker, you'll need to make sure that your cross-seed container has read access to the data directories of your torrents, mapped to the same path as rTorrent. In order for cross-seed to prove to rTorrent that a torrent is completed, it must check the modification timestamps of all the torrent's files.

**Comment by adamwinn** [comment]
From `sudo /usr/local/bin/docker inspect rutorrent-binhex-LTS` I have
```
"Binds": [
                "/volume1/Torrents:/Torrents:rw",
                "/volume1/docker/rutorrent-binhex-LTS:/config:rw",
                "/volume1/Torrents:/volume1/Torrents:rw",
                "/volume1/LTS-Movies-MZ:/LTS-Movies-MZ:rw",
                "/volume1/LTS-Movies-AL:/LTS-Movies-AL:rw",
            ],
```

I use [autotorrent](https://github.com/JohnDoee/autotorrent) and it adds it with fast resume just fine so I assume my setup is OK.

I'm guessing this might be related to the debug output I see when it adds the torrents:
`File /Torrents/LTS-Movies-MZ/xxxxx/VIDEO_TS/VIDEO_TS.BUP either doesn't exist or is the wrong size.`

**Comment by mmgoodnow** [comment]
What are the binds for cross-seed?

It looks like CS is looking for a file at `/Torrents/LTS-Movies-MZ/xxxxx/VIDEO_TS/VIDEO_TS.BUP`. For that path to resolve, I think you'd need a symlink at `/volume1/Torrents/LTS-Movies-MZ` to point to `/LTS-Movies-MZ`

**Comment by mmgoodnow** [comment]
Any updates here? If not I'll assume you figured out

**Comment by adamwinn** [comment]
I got super busy and haven't messed with this since

**Comment by mmgoodnow** [comment]
I've got a hunch that this is the same as #101 so I'll close it out. If you encounter it again after the fix for #101 goes in then please reopen

**Comment by adamwinn** [comment]
All of my torrents had already been downloaded so I doubt it's the same.  All I remember is the exact same paths/setup worked fine for Autotorrent but not cross-seed  

My nas was down for a month during a raid6 rebuild.  I can try this again in a bit
